### PR TITLE
adding scripts to installing chef and puppet by cloud-init. Update th…

### PR DESCRIPTION
**Pull request by hmunfru**
Scripts to be executed in booting time by cloud-init
It includes the gpg and support key
It updates the user data
@jframos @chemaper

### CODE REVIEW

File: src/assembly/jetty/resources/userdata [hunk 1]
```diff
@@ -14,70 +14,21 @@
...
-  sudo: ALL=(ALL) NOPASSWD:ALL
+chefkey: |
+{validation_key}
+
+fiware-support:
+   sshkey: {ssh_key}
 
-ssh_pwauth: False
+   gpgkey: |
+      {gpg_key}
```
Comment (jframos) [inline src/assembly/jetty/resources/userdata]: PaaSManager does not support the replacement of these parameters. The current implementation only works with the parameter _{support_key}_. I think this userdata should be the final userdata for this version of PaaSManager. Should we implement the code to manage the replacement of these vars or should we put fixed data for that?

File: src/assembly/jetty/resources/userdata [hunk 2]
```diff
@@ -14,70 +14,21 @@
...
 
 # Start daemonized chef-client and DEM monitoring
 # Key from http://apt.opscode.com/packages@opscode.com.gpg.key
 runcmd:
- - OHAI_TIME_DIR="$(find / -name ohai_time.rb)"
- - sed -i 's/ohai_time Time.now.to_f/ohai_time Time.now/' ${OHAI_TIME_DIR}
- - chef-client -d -i 60 -s 6 -L /var/log/chef/client.log
- - curl -L -s -k https://xifisvn.esl.eng.it/wp3/software/DEM_Adapter/install.sh | bash
```
Comment (jframos) [inline src/assembly/jetty/resources/userdata]: You cannot remove the installation of this monitoring component.

File: src/assembly/jetty/resources/installconflang.sh [hunk 3]
```diff
@@ -0,0 +1,192 @@
+#!/bin/sh
```
Comment (flopezag) [inline src/assembly/jetty/resources/installconflang.sh]: HEADER...

File: src/assembly/jetty/resources/user_data_script.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python
```
Comment (flopezag) [inline src/assembly/jetty/resources/user_data_script.py]: HEADER...

File: src/assembly/jetty/resources/userdata [hunk 5]
```diff
@@ -14,70 +14,21 @@
```
Comment (flopezag) [inline src/assembly/jetty/resources/userdata]: HEADER...?

**Comment by jframos** [comment]
Add some documentation in each implemented procedure.

File: src/assembly/jetty/resources/userdata (hunk 1 above)
Comment (hmunfru) [inline src/assembly/jetty/resources/userdata]: Hi
Curretly, the gpg_key will be written manually in the file. I don't want to upload it. In future releases, the paas manger will change.

File: src/assembly/jetty/resources/installconflang.sh (hunk 3 above)
Comment (hmunfru) [inline src/assembly/jetty/resources/installconflang.sh]: Done

File: src/assembly/jetty/resources/user_data_script.py (hunk 4 above)
Comment (hmunfru) [inline src/assembly/jetty/resources/user_data_script.py]: Done

File: src/assembly/jetty/resources/installconflang.sh (hunk 3 above)
Comment (flopezag) [inline src/assembly/jetty/resources/installconflang.sh]: NOP, you have to include the Telefonica header to all the files.

File: src/assembly/jetty/resources/user_data_script.py (hunk 4 above)
Comment (flopezag) [inline src/assembly/jetty/resources/user_data_script.py]: NOP, you have to include also the Telefonica header to all the files.

Comment (hmunfru) [inline src/assembly/jetty/resources/user_data_script.py]: ah.. the license. OK!

**Comment by jframos** [comment]
LGTM

**Comment by chemaper** [comment]
LGTM

**Pull request merged**